runtime/foucoco/src/assets.rs

```diff
@@ -3,6 +3,7 @@
 pub mod xcm_assets {
 	use runtime_common::create_xcm_id;
 	create_xcm_id!(RELAY, 0);
+	create_xcm_id!(MOONBASE_DEV, 2);
```

Comment:
Should we just use `1`? Or is there a reason you chose `2`?

Reply:
I chose `2` because I thought that if we need to register some new asset on Rococo Foucoco in the future to have that asset to be `1`

Reply:
Or we could just use `1`, let me know what you think

Reply:
Let's just go with `1`, if that's not yet assigned to something else. (And as far as I know it's not)